core/controllers/improvements.py

```diff
@@ -113,14 +113,11 @@ def get(self, exploration_id):
 
 
 class ExplorationImprovementsConfigHandler(base.BaseHandler):
-    """Handles fetching the configuration of exploration tasks.
-
-    NOTE: Only exploration creators and editors can interface with tasks.
-    """
+    """Handles fetching the configuration of exploration tasks."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
-    @acl_decorators.can_edit_exploration
+    @acl_decorators.can_play_exploration
```

Comment:
Why relax this?

Reply:
Because these options configure the UX directly, rather than offer a means to read/write tasks. To me, it makes more sense for such a configuration to be less constrained so that we can always have the values available (rather than require permissions checks on the frontend).

I'm not 100% committed to the idea, and could be convinced pretty easily to change it back.

Reply:
APIs should be scoped to the correct context, and the context here is clearly the editor page. Future developers may assume this and add information here that is only meant to be for editors.

So I would suggest changing it back, and not having clients make assumptions about the values of the stuff that's exposed here in the first place.

Reply:
Done, have just changed it back.